src/card/aggregate-status/aggregate-status-card.component.js

```diff
@@ -103,28 +112,49 @@
           "href":"#"
         }
       ]
-     };
+    };
+
```

Comment:
Should setup the $scope.aggStatusAlt2 here with the title but the count and notifications being unknown.